fluent/test/tags_test.js

```diff
@@ -1,82 +0,0 @@
-'use strict';
-
-import assert from 'assert';
-
-import { MessageContext } from '../src/context';
```

Comment:
Would you like me to land a follow-up which removes tags from `MessageContext`'s resolver?